packages/datadog-plugin-express/test/integration-test/server.mjs

```diff
@@ -1,8 +1,6 @@
-import tracer from 'dd-trace'
+import 'dd-trace/init.js'
```

Comment:
This can be changed to:

```js
import 'dd-trace/init'
```

I think the linter might not be parsing .mjs files

Reply:
Actually, I think I'm wrong, since we're loading from an .mjs file.